test/integration/test-video-players-helper.js

```diff
@@ -156,8 +156,7 @@ export function runVideoPlayerIntegrationTests(
       });
     });
 
-    // TODO(cvializ, #10283): Find out why this is flaky on master.
-    it.skip('should trigger pause analytics when the video pauses', function() {
```

Comment:
Was some other change landed earlier today that would make this test more reliable? I'm afraid that simply unskipping these tests will make master red all over again.

Failures:
https://travis-ci.org/ampproject/amphtml/jobs/343908930#L674
https://travis-ci.org/ampproject/amphtml/jobs/343912508#L677

Reply:
@cvializ @aghassemi There are still more video related integration test failures on `master`. This is after skipping the failing tests from this morning. See https://travis-ci.org/ampproject/amphtml/jobs/344022650#L671

Is there any way these tests can be made hermetic (or at least more reliable) when run from Travis?

Reply:
Piling on one more comment :)

See https://github.com/ampproject/amphtml/issues/10974#issuecomment-323202133

Reply:
I inferred, maybe incorrectly, that the 3q video player was the source of most of the flakiness, but I didn't dig through the old logs to verify. In what way are the amp-video tests non-hermetic? It should be loading the video files locally, not across the network.

Reply:
Never mind, it's likely that I incorrectly (?) inferred what this PR was doing.

I take it that by skipping all the `amp-3q-video` tests in `test/integration/test-video-players.js`, you're implicitly skipping all of its variants declared in `test/integration/test-video-players-helper.js`, correct? If so, this PR LGTM and I retract my earlier comments.

Reply:
That's correct. But also I would be interested to know if the video tests are non-hermetic in some way I didn't realize, because you did highlight a build that failed because of the amp-video implementation and not the amp-3q-video : /

Reply:
Might be worth running the tests locally in watch mode and looking at the chrome dev tools logs.

```
gulp test --integration --watch --files <test_file>
```

Reply:
I am curious why these became flaky all of a sudden ( I was under the wrong impression that it was just about `amp-3q-video` which has a history of breaking on us due to their downtime ). These tests have been stable for a long time and nothing with videos recently changed. Might be related to a new version of Chrome that just got released or something with Travis that changed.

Reply:
also looks like integration tests passed on this PR. If we close/open PR 5 times to run CI and it does not flake, is that good enough to merge @rsimha-amp ?

Reply:
No need to close / reopen a PR if you just want to re-run just the integration tests. You can merely hit "Restart job" for the `integration_tests` shard / job at https://travis-ci.org/ampproject/amphtml/jobs/344038032.

I'm not sure that's necessary though, because we run `gulp dist` for push builds on `master` while we run `gulp build` for PR builds. (This is due to how long `gulp dist` takes.)

I'd say just merge this and see how it goes.

`</We'll do it live!>`

Reply:
Master is still red, so I'm going to merge this on your behalf to make it green.